nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -158,6 +158,7 @@ let
           server_name ${serverName} ${concatStringsSep " " vhost.serverAliases};
           ${acmeLocation}
           ${optionalString (vhost.root != null) "root ${vhost.root};"}
+          ${optionalString (vhost.alias != null) "alias ${vhost.alias};"}
```

Comment:
Huh; I thought I tested it, but I must have missed something.  That bit is now removed.